scripts/xdist/prepare_xdist_nodes.sh

```diff
@@ -0,0 +1,15 @@
+python scripts/xdist/pytest_container_manager.py -a up -n ${XDIST_NUM_CONTAINERS} \
+-t ${XDIST_CONTAINER_TASK_NAME} \
+-s ${XDIST_CONTAINER_SUBNET} \
+-sg ${XDIST_CONTAINER_SECURITY_GROUP}
+
+ip_list=$(<pytest_container_ip_list.txt)
+
+for ip in $ip_list
+do
+    cmd=$cmd"ssh ubuntu@$ip 'cd /edx/app/edxapp/edx-platform; git pull -q; git checkout -q ${XDIST_GIT_BRANCH}; source /edx/app/edxapp/edxapp_env; pip install -qr requirements/edx/development.txt' & "
```

Comment:
Also, why are you installing development? Shouldn't it be testing.txt or jenkins.txt, according to OEP-18?